tests/utils/stl/test/tests.py

```diff
@@ -71,6 +71,10 @@ def configureTest(self, litConfig):
             return Result(UNSUPPORTED, "Test does not require any of the features specified in limit_to_features: %s" %
                           msg)
 
+        if 'edg_only' in self.config.available_features:
+            if not 'edg' in self.requires:
+                return Result(UNSUPPORTED, 'We only run /BE tests with the edg_only flag')

```

Comment:
Let's go ahead and combine these conditions. (Also, I'm super pedantic about "only" and I'll use this change as an excuse to fix the pre-existing occurrence, too!)
```suggestion
        if 'edg_only' in self.config.available_features and 'edg' not in self.requires:
            return Result(UNSUPPORTED, 'We run only /BE tests with the edg_only flag')
```